### DISCUSSION TITLE
Why is SYSTEM_UPGRADE_JOB_KUBECTL_IMAGE in0.8.1 so old rancher/kubectl down at 1.18

### DISCUSSION THREAD

**Discussion by Jackbennett** [rancher/system-upgrade-controller#181] [Q&A] [answered]
doesn't kubectl support prior versions so I should set this up to 1.23 to upgrade my 1.20 k3s nodes?

**Comment by dweomer** [comment, marked as answer]
Rancher has to support some older k8s versions and so I settled on 1.18, knowing it would be likely run on 1.16.x systems.
